dgbpy/torch_classes.py

```diff
@@ -501,12 +504,21 @@ def compute_loss_func(self):
     def one_batch(self, i, input, target):
         try:
             self.iter = i
-            self.input, self.target = input, target ;   self('begin_batch')
-            self.out = self.model(self.input) ;         self('after_pred')
-            self.compute_loss_func() ;                  self('after_loss')
-            if not self.in_train: return
-            self.loss.backward() ;                      self('after_backward')
-            self.optimizer.step() ;                     self('after_step')
+            self.input, self.target = input, target ;                               self('begin_batch')
+            if self.tofp16:
+                with torch.cuda.amp.autocast(enabled=torch.cuda.is_available()):
+                    self.out = self.model(self.input) ;                             self('after_pred')
+                    self.compute_loss_func() ;                                      self('after_loss')
+                    if not self.in_train: return
+                self.gradScaler.scale(self.loss).backward();                        self('after_backward')
```

Comment:
the line becomes too long, can you please reduce the indent?

Reply:
Thank @arnauddgbes , will do